src/exprestive.coffee

```diff
@@ -80,20 +86,35 @@ class Exprestive
     @routesMethod @getRoutesHelperMethods()
 
 
+  # Save a function to @reversePaths to get a url back from a route name
+  registerReverseRoute: ({routeName, url}) ->
+    return unless routeName?
+    @reversePaths[camelCase routeName] = (id) ->
+      url.replace ':id', id
+
+
   # A helper method for automatically binding restful controllers in a routes file
   # This is passed as "resources" to the routes file function parameter hash
   resourcesHelperMethod: (controllerName) =>
-    resourceRoutes = [
-      [ 'GET',    "/#{controllerName}",          'index'   ]
-      [ 'GET',    "/#{controllerName}/new",      'new'     ]
-      [ 'POST',   "/#{controllerName}",          'create'  ]
-      [ 'GET',    "/#{controllerName}/:id",      'show'    ]
-      [ 'GET',    "/#{controllerName}/:id/edit", 'edit'    ]
-      [ 'PUT',    "/#{controllerName}/:id",      'update'  ]
-      [ 'DELETE', "/#{controllerName}/:id",      'destroy' ]
+    singular = singularize controllerName
+    plural = pluralize controllerName
+    for [ httpMethod ,  url                   , controllerAction , routeName          ] in [
```

Comment:
If you don't pad everything with spaces here, it takes up less horizontal space, and looks more like normal CoffeeScript.